src/net_processing.cpp

```diff
@@ -4457,22 +4668,26 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
             peer->m_blocks_for_inv_relay.clear();
 
             if (pto->m_tx_relay != nullptr) {
+                const bool supports_recon = m_reconciliation.IsPeerRegistered(pto->GetId());
                 LOCK(pto->m_tx_relay->cs_tx_inventory);
                 // Check whether periodic sends should happen
                 bool fSendTrickle = pto->HasPermission(PF_NOBAN);
                 if (pto->m_tx_relay->nNextInvSend < current_time) {
                     fSendTrickle = true;
-                    if (pto->IsInboundConn()) {
+                    if (!supports_recon && pto->IsInboundConn()) {
                         pto->m_tx_relay->nNextInvSend = m_connman.PoissonNextSendInbound(current_time, INBOUND_INVENTORY_BROADCAST_INTERVAL);
                     } else {
+                        // Use half the delay for outbound peers, as there is less privacy concern for them.
+                        // Also use half delay for inbound peers if they use reconciliations,
+                        // because we have an additional delay for sending out requested sketches.
```

Comment:
Not sure if the following change on how reconciliation is influencing broadcast interval is covered by `p2p_erlay.py`

```
diff --git a/src/net_processing.cpp b/src/net_processing.cpp
index 1818fc0f2..28641f576 100644
--- a/src/net_processing.cpp
+++ b/src/net_processing.cpp
@@ -4664,7 +4664,7 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                 bool fSendTrickle = pto->HasPermission(PF_NOBAN);
                 if (pto->m_tx_relay->nNextInvSend < current_time) {
                     fSendTrickle = true;
-                    if (!supports_recon && pto->IsInboundConn()) {
+                    if (pto->IsInboundConn()) {
                         pto->m_tx_relay->nNextInvSend = m_connman.PoissonNextSendInbound(current_time, INBOUND_INVENTORY_BROADCAST_INTERVAL);
                     } else {
                         // Use half the delay for outbound peers, as there is less privacy concern for them.
```